apps/server/src/modules/school/mapper/school.mapper.ts

```diff
@@ -0,0 +1,26 @@
+import { SchoolDto } from '@src/modules/school/uc/dto/school.dto';
+import { School } from '@shared/domain';
+
+export class SchoolMapper {
+	static mapToEntity(schoolDto: SchoolDto): School {
+		const school = new School({
+			name: schoolDto.name,
+		});
+		if (schoolDto.id) {
+			school.id = schoolDto.id;
+		}
+		return school;
+	}
+
+	static mapEntityToEntity(target: School, source: School): School {
```

Comment:
The purpose is the patching of an entity. But in this layer it is only a mapping and can be extended in the future. We created a function patchSchool in the service which calls this method.

Reply:
We prepared this function and the whole module for the future. The easier way would to call everything directly but then the code would "grow historically".
I agree with you at the moment it is a little bit over engineered but 
I think we will not only change the name, but also the other fields like features or language and so on.

No, its only a simple mapping and there shouldn't be any business logic inside a mapper.
I don't know if it would be better to have a patch function for every field inside the service. Could be a point for the architecture meeting.